app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+		   2008-2010 Jonathan Beck
+		   2009 Nikias Bassen
+		   2008 Matt Colyer
+		   "
+LICENSE="GNU GPL v2
+		 GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/$portName-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:plistutil$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist++$secondaryArchSuffix = $portVersion compat >= 0
```

Comment:
BTW, when you list the contents of the devel package with **`package list /path/to/libplist_devel-*.hpkg`** you can see what the library names are. You can also have a look at the **`configure.ac`** and notice that there is a "3.0.0" somewhere.